runtime/kusama/src/weights/frame_benchmarking_baseline.rs

```diff
@@ -46,46 +46,46 @@ pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_benchmarking::baseline::WeightInfo for WeightInfo<T> {
 	/// The range of component `i` is `[0, 1000000]`.
 	fn addition(_i: u32, ) -> Weight {
-		Weight::from_ref_time(114_000 as u64)
+		Weight::from_ref_time(105_000 as u64)
 	}
 	/// The range of component `i` is `[0, 1000000]`.
 	fn subtraction(_i: u32, ) -> Weight {
-		Weight::from_ref_time(125_000 as u64)
+		Weight::from_ref_time(112_000 as u64)
```

Comment:
@koute @ggwpez strange to me that these end in `000`?

What are the chances if the precision is down to that

Reply:
I think the increased precision only applied to the linear component, or @koute ?

Reply:
that makes sense

Reply:
Correct, the increased precision is only for the component weights, as that's where it can make a difference.